packages/graphql/src/stitching/remoteSchemaWrapped.ts

```diff
@@ -65,13 +68,16 @@ export const remoteSchemaWrapped = wrapSchema<DataContext>({
       // When we respond eagerly with a result, but receive an updated value
       // for the query, we can "push" the data down using the pushFragment subscription
       onUpdatedResult (result) {
-        obj.context?.graphql.pushResult({
+        context.graphql.pushResult({
           result: result?.[info.fieldName] ?? null,
           source: obj.rootValue,
           info,
-          ctx: obj.context,
+          ctx: context,
         })
       },
+      invalidateCache () {
```

Comment:
Not super happy with the naming of this, might change to `onInvalidateCache`, and centralize the client & server invalidation calls as like `ctx.invalidateGraphQLCache()`.